docs/content/en/providers/spotify.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Spotify
+description: This provider is based on oauth2 scheme and supports all scheme options
+position: 36
+category: Providers
+---
+
+[Source Code](https://github.com/nuxt-community/auth-module/blob/dev/src/providers/spotify.ts)
+
+## Usage
+
+```js
+auth: {
+  strategies: {
+    spotify: {
+      clientId: '...',
+      clientSecret: '...'
+    },
+  }
+}
+```
+
+Anywhere in your application logic:
+
+```js
+this.$auth.loginWith('spotify')
+```
+
+💁 This provider is based on [oauth2 scheme](../schemes/oauth2) and supports all scheme options.
+
+## Obtaining `clientId` and `clientSecret`
+
+This option is **REQUIRED**. To obtain one, create your app in the [Spotify Developer Dashboard](https://developer.spotify.com/dashboard/) and use provided "Client ID" and "Client Secret".
```

Comment:
Can you please add a section to the bottom of the README under "Development" with instructions on how to get Spotify working in local development?

Reply:
I placed it at the top, so you can see it directly. The usage is the same.

Reply:
I don't see any changes to the README?

Reply:
(Spoiler: I'm going to follow the directions you write in the README to set up and test this before merging :) )

Reply:
Yeah sure.